assets/airports/essa.json

```diff
@@ -0,0 +1,2865 @@
+{
+    "_comment": "Source airport info: https://www.aro.lfv.se/Editorial/View/7952/ES_AD_2_ESSA_en",
+    "airac": 2101,
+    "radio": {
+        "twr": "Arlanda Tower",
+        "app": "Stockholm Control",
+        "dep": "Stockholm Control"
+    },
+    "icao": "ESSA",
+    "iata": "ARN",
+    "magnetic_north": 6,
+    "ctr_radius": 150,
+    "ctr_ceiling": 9500,
+    "_comment": "Initial climb clearance is 3000 ft if not on a SID, all SIDs have 5000 ft",
+    "initial_alt": 3000,
+    "position": ["N59d39m07.00", "E017d55m07.00", "138ft"],
+    "rangeRings": {
+        "enabled": true,
+        "center": ["N59d39m07.00", "E017d55m07.00"],
+        "radius_nm": 5.0
+    },
+    "has_terrain": false,
+    "wind": {
+        "angle": 180,
+        "speed": 5
+    },
+    "arrivalRunway": "19L",
+    "departureRunway": "19R",
+    "defaultMaps": ["Arlanda CTR", "Stockholm TMA", "Sector Uppsala + Uppsala CTR", "Sector Västerås + Västerås CTR", "Bromma CTR"],
+    "_comment": "Airspaces below are simplifications of Stockholm TMA and Arlanda CTR where adjacent sectors are combined where possible. All real sectors are drawn in defaultMaps for realistic looks.",
+    "airspace": [
+        {
+            "_comment": "Arlanda CTR West + Arlanda CTR East + Overlapping part of Stockholm TMA b",
+            "floor":          0,
+            "ceiling":        9500,
+            "airspace_class": "C",
+            "poly":
+            [
+                ["N59d48m58.00", "E017d46m48.00"],
+                ["N59d48m58.00", "E017d59m56.00"],
+                ["N59d48m58.00", "E018d10m47.00"],
+                ["N59d46m28.00", "E018d15m17.00"],
+                ["N59d37m58.00", "E018d19m17.00"],
+                ["N59d29m28.00", "E017d57m48.00"],
+                ["N59d29m54.00", "E017d53m02.00"],
+                ["N59d30m28.00", "E017d46m48.00"],
+                ["N59d33m28.00", "E017d41m48.00"],
+                ["N59d48m58.00", "E017d46m48.00"]
```

Comment:
Do I need to close the polygon with the starting point? This applies to all polygons.

Reply:
I looked this up myself and have removed the repetition of the first coordinate. Will push soon.